LBH.AdultSocialCare.Api/V1/Infrastructure/Entities/Common/CarePackage.cs

```diff
@@ -50,5 +50,6 @@ public class CarePackage : BaseEntity
         public virtual ICollection<CarePackageDetail> Details { get; set; }
 
         public virtual ICollection<CarePackageReclaim> Reclaims { get; set; }
+        public virtual ICollection<CarePackageHistory> CarePackageHistories { get; set; }
```

Comment:
let's keep naming uniform - use Histories or HistoryEntries, since we're already in context of CarePackage